features/admin.identity-providers.v1/components/utils/saml-idp-utils.ts

```diff
@@ -333,3 +333,19 @@ export const isUrl = (value: string): string => {
         ? undefined
         : "This URL is invalid.";
 };
+
+export const DEFAULT_NAME_ATTRIBUTE_FORMAT: string = "urn:oasis:names:tc:SAML:2.0:attrname-format:basic";
```

Comment:
shall we move this to constants/ ?

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/7549/commits/57cd3eea6be5b819740cd18011486fa882db1108